Add support for Huion H580X (#2040)
OpenTabletDriver/OpenTabletDriver: TABLETS.md

| XP-Pen Star G640S             |     Supported     |
| XP-Pen Star G960S             |     Supported     |
| XP-Pen Star G960S Plus        |     Supported     |

> | Huion H580X                   |    Has Quirks     | User may have to replug their tablet in order for detection to occur.

Reviewer: ```suggestion
| Huion H580X                   |    Has Quirks     | User may have to replug their tablet until it is detected.
```
